.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Generally, it is a good practice to use predefined templates for `.gitignore` file. But, we can update this file later.